src/main/kotlin/no/nav/syfo/application/SenOppfolgingService.kt

```diff
@@ -71,6 +71,7 @@ class SenOppfolgingService(
     fun publishUnpublishedKandidatStatus(): List<Result<SenOppfolgingKandidat>> {
         val unpublishedKandidatStatuser = senOppfolgingRepository.getUnpublishedKandidatStatuser()
         return unpublishedKandidatStatuser
+            .filter { kandidat -> kandidat.svar != null || kandidat.isVarsletForMinstTiDagerSiden() }
```

Comment:
Mister vi ikke publiseringen av vurderingene her?

Reply:
Hm, tror ikke logikken slik den var publiserte vurderinger uten at kandidaten hadde svart.
Vil vi ikke få samme resultat som med sqlen (bare inkludert de som er varslet for 10 dager siden i tillegg)? Siden vi hadde `svar_at IS NOT NULL AND  ( .. sjekk på published_at eller vurdering..  )` ?

Reply:
Men hvis det skal være mulig gjøre en vurdering før kandidaten har svart eller det har gått 10 dager siden varsel så må vi endre på dette.